client/packages/common/src/ui/layout/tables/components/Cells/NumberInputCell/NumberInputCell.tsx

```diff
@@ -30,17 +30,37 @@ export const NumberInputCell = <T extends RecordWithId>({
   endAdornment,
   error,
   slotProps,
+  debounce = 250,
 }: CellProps<T> &
   NumericInputProps & {
     id?: string;
     TextInputProps?: StandardTextFieldProps;
     endAdornment?: string;
     error?: boolean;
-  } & Pick<StandardTextFieldProps, 'slotProps'>): React.ReactElement<
-  CellProps<T>
-> => {
+  } & Pick<StandardTextFieldProps, 'slotProps'> & {
+    debounce?: number;
```

Comment:
customisable debounce added, not sure .25sec is long enough when typing on tablet?